### PR TITLE
Demo for issue #635

### PR DESCRIPTION
**Pull request by natac13**
#635 

however, I have realized my error.
I was follow the epic stack in my remix app and was not ready for Sentry. Therefore the comment about it in `server/index.ts` made me this I could get rid of the NODE_ENV check. Therefore, I was left with just the `await getBuild()` call. However, because I was calling `getBuild` now in development the `viteDevServer.ssrLoadModule` part needed to be wrapped in a function like the remix/website example

To summarize. If you try this branch out and remove the wrapper function around `viteDevServer.ssrLoadModule` you will see that the app starts but updates inside a loader are not reflected. Which is where I thought to help

Please let me know if you would like more. Thank you for the Epic Stack and all your work

## Checklist

- [ ] Tests updated
- [ ] Docs updated

### CODE REVIEW

**Comment by natac13** [comment]
And just curious if your @ts-ignore is just for comment? Since it does not appear to be needed to avoid issues?

Screenshot 2024-03-04 at 1 37 13 PM

**Comment by kentcdodds** [comment]
On the `@ts-ignore` bit, the `#build/server/index.js` file only exists after you run the build. But before you run the build, TypeScript will not be able to find it. That's why we have the `@ts-ignore`.

So if I understand you correctly, we don't need to make any changes because you mistakenly removed the `production` check when that actually has nothing to do with Sentry and actually is what supports HMR in the first place.

Thanks!

**Comment by natac13** [comment]
Yep. I made the mistake

**Comment by ajwtech** [comment]
I am not sure what just happened but either Github just changed something, or something related to this PR just broke the github extension in vscode. 
image
I use this to walk me through any PR's that are closed so I can to decide if I take them. I was just working on the Vite integration and then  this error started coming up. Any ideas? This is an odd one. But if anyone is curious this happens when I click on "load more" for a custom query "is closed"
